null_safety_examples/misc/test/library_tour/core_test.dart

```diff
@@ -233,10 +233,10 @@ void main() {
       fruits.clear();
       assert(fruits.isEmpty);
 
-      // You can also create a List using one of the constructors.
+      // You can also create a List using one of the factory constructors.
       var vegetables = List.filled(99, 'broccoli');
+      assert(vegetables.every((v) => v == 'broccoli'));
       // #enddocregion List
-      assert(vegetables.isEmpty);
```

Comment:
I'm a bit worried now. How did this pass the tests? (Not a question for you, @parlough, I'm just asking the universe I guess.)